test/browsers/devicefarm.test.ts

```diff
@@ -1,36 +1,41 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 import './wdio-mock';
-import DeviceFarmClientMock from 'aws-sdk/clients/devicefarm';
```

Comment:
Refactored this test to use explicit `jest.mock()` instead of implicit `__mocks__/aws-sdk/clients/devicefarm.js` file